package.json

```diff
@@ -74,28 +72,37 @@
     "typescript": "5.7.3",
     "winston": "^3.10.0"
   },
-  "resolutions": {
-    "//": "we need to pin wide-align/string-width because v5 & up are ESM only",
-    "wide-align/string-width": "^4.2.0",
-    "node-gyp": "^10.0.0"
+  "pnpm": {
+    "overrides": {
+      "a": "we need to pin wide-align/string-width because v5 & up are ESM only",
```

Comment:
`pnpm` was complaining about the `//` resolutions when I migrated to `overrides`. Using an alphanumeric package name for the comment appears to work but feels pretty gross. 

pnpm does support a package.json that includes comments (`package.json5`) but from what I've read not all other JS tooling does so that feels like a deal-breaker. 

Do we still need these overrides at all?

Reply:
I'm leaning that way too! Git should have details as to why the line was added even if it takes a bit of digging.

Reply:
Could have a separate section for commenting that is custom of `packageName -> comment` and then have a test that verifies there's an override per comment and vice versa? Seems like a lot of work for now...so still...just delete them?

Reply:
> I'm leaning that way too! Git should have details as to why the line was added even if it takes a bit of digging.

This is harder than you might think and generally, doesn't happen unless someone absolutely needs to know. We have had this issue in the past quite a bit (why was this resolution there? Let me just get rid of it. Oh I can fix this error...) wasting lots of time when a comment right there would've said "we tried all that, just do this...see this ticket for info". Finding the right commit in Git can be a pain sometimes as code changes and moves around. Having the comments colocated was super useful and time-saving.

Reply:
Ok, I brought it back with a pattern I think works. Install works which I think is the only place that'd parse that section of `package.json`. I do wonder if our current deps need that override, but I'm paranoid to remove it and how to fully verify.